src/inventory.cpp

```diff
@@ -964,15 +964,13 @@ InventoryList * Inventory::addList(const std::string &name, u32 size)
 {
 	setModified();
 
-	// Remove existing lists
+	// Reset existing lists
```

Comment:
maybe add a comment that it's important not to require the resize lock here

Reply:
Only inventory actions (which make use of Inventory/InventoryList) claim and release resize locks. This function still checks against that inside `list->setSize(size);`, which is the proper way to do.

Reply:
yes